### ISSUE TITLE
Payment method list and re-ordering accessibility

### ISSUE THREAD

**Issue by mikejolley** [woocommerce/woocommerce#13624]
See https://github.com/woocommerce/woocommerce/issues/13288 for context. cc @jameskoster 

Users can drag and drop payment methods to define how they appear on the frontend. We should allows these to be sorted using buttons/icons like the menus in the customizer.

**Comment by jameskoster** [comment]
I haven't tested but anything drag and drop is likely to be an issue on touch devices as well.

Re-order button with arrows works for me. Similar to Menus in the Customizer.

**Comment by jameskoster** [comment]
If we have the icons, we don't need a number input :)

**Comment by Uriel1339** [comment]
If you have a payment on the bottom and stripe for example adds like 10 methods it's just a lot of clicking to get it up to the top. Unless it doesn't save & refresh each time you click 'up' until you hit the save/update button.

That is my main concern. Also there should be a tool tip somewhere that states that the order is reflected on the checkout screen.

**Comment by mikejolley** [comment]
See https://github.com/woocommerce/woocommerce/pull/19844
